inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -72,13 +73,25 @@ def simulate(
 
             nsteps_minim: Number of steps for energy minimization.
         """
+
+        #Compute charge if it is not provided
+        if self.charged is None:
+            self.charged = self.compute_charge()
+
+        #Edit commands.json according to the charge of the protein
+        commands_path = os.path.join(self.template_dir, "commands.json")
+        replace_params_in_template(self.template_dir, "commands.json.jinja", {
+            "warnings": "-maxwarn 1",
+            "solvents": "SOL"
+        }, commands_path)
```

Comment:
Why do you need to replace these parameters in a jinja template if they are constant?